controllers/subscriber/README.md

```diff
@@ -135,25 +135,16 @@ An example of what this could look like altogether:
 </cable-subscription>
 <younger-viewer for="rogers"></younger-viewer>
 ```
+NOTE: If wanting to use IdSubscriberController, you must call its `hostPropertiesChanged` function in the `updated` function, passing along the `changedProperties` parameter:
```

Comment:
Added this note here for the IdSubscriberController to remember to call the hostPropertiesChanged function in the updated function.